compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUSelectLoweringStrategy.cpp

```diff
@@ -90,8 +91,24 @@ verifyEntryPoint(ModuleOp moduleOp,
     for (auto [index, attr] : llvm::enumerate(workgroupSizeAttr.value())) {
       workgroupSizes[index] = llvm::cast<IntegerAttr>(attr).getInt();
     }
-    return verifyLoweringConfiguration(moduleOp, translationInfo,
-                                       workgroupSizes, verifyGPUMatmulPipeline);
+    std::optional<int64_t> subgroupSize;
+    if (auto subgroupSizeAttr = exportOp.getSubgroupSize()) {
```

Comment:
Just pointing out that this logic is problematic when I move this op to work on just `FunctionOpInterface`. Basically we might have a funcOp that is not the function referenced in the `exportOp`. So this tight coupling needs to be undone at some point.